main.py

```diff
@@ -73,9 +75,11 @@ def draw_help(self):
             g.WIN.blit(
                 self.close_text,
                 (
-                    (3 * g.WIDTH + g.FRAME_GAP * 3 - 2 * self.close_text.get_width())
+                    (3 * g.WIDTH + g.FRAME_GAP * 3 -
+                     2 * self.close_text.get_width())
                     // 4,
-                    (g.HEIGHT * 0.5 - g.FRAME_GAP * 1.5 - self.close_text.get_height())
+                    (g.HEIGHT * 0.5 - g.FRAME_GAP *
+                     1.5 - self.close_text.get_height())
```

Comment:
Why this change?

Reply:
Fixes inconsistent formatting of the code. It was just for the uniformity of the code. 
Same reason for the rest.

Reply:
I don't see the inconsistency you mention and your changes aren't pep8 compliant. 

The title of your change is also for something else entirely so if you want to make any changes to the formatting of the code you should open a different PR for that.

Reply:
Oh Okay, I Will make the code pep8 compliant and open a new PR for that.

Reply:
For this PR you should focus on adding text to show the current player's turn like the subject says so remove every commit that has nothing to do with that.

Reply:
Okay, I will Do so

Reply:
This PR got really messy, so I will be closing this PR and creating a new clean PR which just includes the necessary commits.